RFS/src/main/java/org/opensearch/migrations/bulkload/version_es_6_8/OpenSearchWorkCoordinator_ES_6_8.java

```diff
@@ -0,0 +1,88 @@
+package org.opensearch.migrations.bulkload.version_es_6_8;
+import java.time.Clock;
+import java.util.function.Consumer;
+
+import org.opensearch.migrations.bulkload.workcoordination.AbstractedHttpClient;
+import org.opensearch.migrations.bulkload.workcoordination.OpenSearchWorkCoordinator;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class OpenSearchWorkCoordinator_ES_6_8 extends OpenSearchWorkCoordinator {    
+        public OpenSearchWorkCoordinator_ES_6_8(
+            AbstractedHttpClient httpClient,
+            long tolerableClientServerClockDifferenceSeconds,
+            String workerId
+        ) {
+            super(httpClient, tolerableClientServerClockDifferenceSeconds, workerId);
+        }
+        public OpenSearchWorkCoordinator_ES_6_8(
+                AbstractedHttpClient httpClient,
+                long tolerableClientServerClockDifferenceSeconds,
+                String workerId,
+                Clock clock
+        ) {
+            super(httpClient, tolerableClientServerClockDifferenceSeconds, workerId, clock);
+        }
+
+        public OpenSearchWorkCoordinator_ES_6_8(
```

Comment:
Can we implement only one of these ctors?

Reply:
One was only being used in tests; I've removed it and replaced it with the constructor we use in the real workflow.